pkg/resourcecreator/resourcecreator_test.go

```diff
@@ -327,6 +327,11 @@ func TestCreate(t *testing.T) {
 						Name: dbName,
 					},
 				},
+				AdditionalUsers: []nais_io_v1alpha1.AdditionalUser{
+					{
+						Name: fmt.Sprintf("%s-%s", instanceName, "read"),
+					},
+				},
```

Comment:
Please avoid adding more tests to resourcecreator_test.go, rather just use the golden file tests only.

Reply:
good ide!